opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -501,6 +507,29 @@ def _transcribe(item, parent, editRate, masterMobs):
     return result
 
 
+def _find_timecode_track_start(track):
+    # See if we can find a starting timecode in here...
+    aaf_metadata = track.metadata.get("AAF", {})
+
+    # Is this a Timecode track?
+    if aaf_metadata.get("MediaKind") == "Timecode":
+        edit_rate = aaf_metadata.get("EditRate", "0")
+        fps = aaf_metadata.get("Segment", {}).get("FPS", 0)
+        start = aaf_metadata.get("Segment", {}).get("Start", "0")
+
+        # Often times there are several timecode tracks, so
+        # we use a heuristic to only pay attention to Timecode
+        # tracks with a FPS that matches the edit rate.
+        if edit_rate == str(fps):
+            return otio.opentime.RationalTime(
+                value=int(start),
+                rate=float(edit_rate)
```

Comment:
what kinds of objects are these?  strings?  just wondering why the cast to float/int